docs/container_types.rst

```diff
@@ -176,33 +176,62 @@ The dictionary terms are used as the chapter :ref:`slug` and the description of
 
 .. note:: Lengthy dictionary terms may be split into more than one chunk.
 
-The ``01.txt`` file contains the description of the term where the header is the title of the term and the rest is the description:
+The ``01.txt`` file contains the description of the term. The term title must always be at the top of the file as a
+h1 heading (a single #). :ref:`Links <linking>` may be used to reference other words, or content in other containers.
+
+.. note:: This type is in progress. We may add Strong's numbers to the markdown as well.
 
 .. code-block:: markdown
 
-    # Aaron
+    # Aaron #
+
```

Comment:
going with the solution you proposed in https://github.com/unfoldingWord-dev/Resource-Container/issues/41#issuecomment-294017481